test/verify/check-storage-stratis

```diff
@@ -36,12 +36,18 @@ class TestStorageStratis(storagelib.StorageCase):
 
         self.stratis_v2 = self.image.startswith("rhel-8") or self.image == "centos-8-stream"
 
+        # the CLI changed in an incompatible way in Fedora 40
+        if '--name' in exe("stratis pool stop --help"):
+            self.stop_type_opt = "--name"
```

Comment:
Maybe you could turn this info `pool_stop_cmd` and have `stratis pool stop` or `stratis pool stop --name` in there, accordingly.  Would be a nice factor-out?

Reply:
Yes, can do. I initially even had this, then changed it to this because I thought "oh, `stratis pool destroy` is going to need that, too", only to find out that it doesn't..